test/domain_bridge/config/topic_options.yaml

```diff
@@ -14,9 +14,11 @@ topics:
       depth: 42
       deadline: 123456
       lifespan: auto
+    remap: bar/remapped
   baz:
     type: test_msgs/msg/Empty
     # Trying other values for deadline and lifespan
     qos:
       deadline: auto
       lifespan: -5
+    remap: ''
```

Comment:
the test should still pass if we delete this line, right?

Reply:
Correct. I added this to test an explicit empty string versus no YAML entry.